frontend/src/components/admin/menu/DictionaryManagement.js

```diff
@@ -272,38 +272,63 @@ function DictionaryManagement() {
     return <TableCell key={cell.id}>{cell.value}</TableCell>;
   };
 
+  // const handleDictionaryMenuItems = (res) => {
+  //   if (componentMounted.current) {
+  //     const selectedItem = dictionaryMenuList.find(
+  //       (item) => item.id === selectedRowId,
+  //     );
+
+  //     console.log("contents of selected item: " + JSON.stringify(selectedItem));
+  //     if (selectedItem) {
+  //       setDictionaryNumber(selectedItem.id);
+  //       setCategory(selectedItem.category);
+  //       setDictionaryEntry(selectedItem.dictEntry);
+  //       setLocalAbbreviation(selectedItem.localAbbreviation);
+  //       setIsActive(yesOrNo.find((item) => item.id === selectedItem.isActive));
+  //       setLastUpdated(selectedItem.lastupdated);
+  //       setOpen(true);
+  //       setEditMode(false);
+  //     }
+  //   }
+  // };
+
   const handleDictionaryMenuItems = (res) => {
     if (componentMounted.current) {
       setDictionaryNumber(res.id);
       setCategory(res.dictionaryCategory);
       setDictionaryEntry(res.dictEntry);
       setIsActive(yesOrNo.find((item) => item.id === res.isActive));
       setLocalAbbreviation(res.localAbbreviation);
-      setLastUpdated(res.lastupdated);
     }
   };
 
   const handleOnClickOnModification = async (event) => {
     event.preventDefault();
     if (selectedRowId) {
-      getFromOpenElisServer(
-        `/rest/Dictionary?ID=${selectedRowId}&startingRecNo=${startingRecNo}`,
-        handleDictionaryMenuItems,
-      );
-
       const selectedItem = dictionaryMenuList.find(
         (item) => item.id === selectedRowId,
       );
+
+      console.log("contents of selected item: " + JSON.stringify(selectedItem));
       if (selectedItem) {
         setDictionaryNumber(selectedItem.id);
-        setCategory(selectedItem.categoryName);
+        setCategory(selectedItem.category);
         setDictionaryEntry(selectedItem.dictEntry);
         setLocalAbbreviation(selectedItem.localAbbreviation);
         setIsActive(yesOrNo.find((item) => item.id === selectedItem.isActive));
-        setLastUpdated(selectedItem.lastupdated); // Set the lastupdated state
+        setLastUpdated(selectedItem.lastupdated);
```

Comment:
i was able to add the lastupdated field but i late realized the old ui actually doesn't send it in the payload when updating an item but rather my issue is we need to populated the `dirtyfields` attribute with changed fields as part of the payload. see you monday 😄